text/0344-conventions-galore.md

```diff
@@ -213,7 +213,7 @@ seem like a minor concern, when we hit 1.0 the lint names will be locked down,
 so it's worth trying to clean them up now.
 
 The basic rule is: the lint name should make sense when read as "allow
-*lint-name*" or "allow *lint-name* items". For example, "allow
+*lint_name*" or "allow *lint_name* items". For example, "allow
```

Comment:
These are just inline examples, but they should follow the `_` convention given in the rest of this section.